README.md

```diff
@@ -126,75 +126,75 @@ The address of the Api3ServerV1 contract. If not specified, the address is loade
 
 The address of the DapiDataRegistry contract.
 
-##### `providers`
+#### `providers`
 
 A record of providers. The record key is the provider name. Provider name is only used for internal purposes and to
 uniquely identify the provider for the given chain.
 
-###### `providers[<NAME>]`
+##### `providers[<NAME>]`
 
 A provider configuration.
 
-`url`
+###### `url`
 
 The URL of the provider.
 
-##### `__Temporary__DapiDataRegistry`
+#### `__Temporary__DapiDataRegistry`
 
 The data needed to make the requests to signed API. This data will in the future be stored on-chain in a
 `DapiDataRegistry` contract. For the time being, they are statically defined in the configuration file.
 
-###### `airnodeToSignedApiUrl`
+##### `airnodeToSignedApiUrl`
 
 A mapping from Airnode address to signed API URL. When data from particular beacon is needed a request is made to the
 signed API corresponding to the beacon address.
 
-###### `dataFeedIdToBeacons`
+##### `activeDapiNames`
+
+An array of dAPI names to execute updates for.
+
+##### `dataFeedIdToBeacons`
 
 A mapping from data feed ID to a list of beacon data.
 
-###### `dataFeedIdToBeacons<DATA_FEED_ID>`
+##### `dataFeedIdToBeacons<DATA_FEED_ID>[n]`
 
 A single element array for a beacon data. If the data feed is a beacon set, the array contains the data for all the
 beacons in the beacon set (in correct order).
 
-`dataFeedIdToBeacons<DATA_FEED_ID>[n]`
```

Comment:
I ran out of nesting options so I just put it in backticks. I think what you did (make it title with #####) is better so I'd keep it.